great-idea/services.html

```diff
@@ -27,13 +27,13 @@
         <a href="#">Features</a>
         <a href="#">About</a>
         <a href="#">Contact</a>
-        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo." />
+        <a href="./index.html"  class="logo"><img src="img/logo.png" alt="Great Idea! Company logo." /></a>
       </nav>
```

Comment:
you can put the image outside a nav tag so the parent will have two children, this will give you more control on the UI , so you can separate easily the elements